projects_test.go

```diff
@@ -116,3 +116,50 @@ func TestAccCreateNonDefaultOrgProject(t *testing.T) {
 		t.Fatalf("Expected new project to not be part of org %s", orgPath)
 	}
 }
+
+func TestAccListProjects(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c := setup(t)
+
+	defer projectTeardown(c)
+
+	rs := testProjectPrefix + randString8()
+
+	u, _, err := c.Users.Current()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	orgPath := "/organizations/" + u.DefaultOrganizationID
+
+	pcr := ProjectCreateRequest{Name: rs}
+	p, _, err := c.Projects.Create(&pcr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if p.Organization.URL != orgPath {
+		t.Fatalf("Expected new project to be part of org %s, not %v", orgPath, p.Organization)
+	}
+
+	listOpt := &ListOptions{
+		Includes: "members",
+	}
+	projs, _, err := c.Projects.List(listOpt)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, proj := range projs {
+		if proj.ID == p.ID {
+			if len(proj.Users) == 0 {
+				t.Fatal("Project users not returned.")
+			}
+
+			if proj.Users[0].FirstName != u.FirstName {
```

Comment:
Maybe test for ID, rather than FirstName. If we, at some point, run the acceptance test with a service account in some CI, the username might be unset (thus same as default for string value in struct), plus it's not unique.